Dockerfile

```diff
@@ -1,26 +1,43 @@
-FROM debian:stable as builder
+FROM alpine:edge as builder
 
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
-        ca-certificates \
-        git \
+RUN apk update && \
+    apk add \
+        gcc \
+        g++ \
+        automake \
+        autoconf \
+        pkgconfig \
+        python2-dev \
         cmake \
-        build-essential \
-        zlib1g-dev \
-        python \
-        wget \
-        xz-utils
+        make \
+        libc-dev \
+        binutils \
+        zlib-dev \
+        libstdc++
 
 RUN mkdir -p /deps
+ARG MAKE_JOBS=-j1
+
+# xml2
```

Comment:
It would be nice to remove this xml dependency entirely. Didn't seem like it was necessary as mentioned [here](https://github.com/zig-lang/zig/issues/853#issuecomment-377496002).

Not anything major, and don't bother removing it just yet since you'll need to redo the llvm/clang build. May as well wait until upstream LLVM puts out the patch release you mentioned and change with everything else, right?